# generic message type for action specification.
builtin_interfaces/Time stamp
string type
string[] keys
string[] values